molecule/default/molecule.yml

```diff
@@ -7,6 +7,7 @@ driver:
 platforms:
   - name: instance
     image: ubuntu:20.04
+    privileged: true
```

Comment:
Is this really required?

Reply:
It is, otherwise we cannot test the bind mount.  Mounting inside the container requires elevated privileges.